sdk/compiler/damlc/tests/daml-test-files/ExplicitDataTypes.daml

```diff
@@ -0,0 +1,20 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- @SUPPORTS-LF-FEATURE DAML_EXPLICIT_DATA_TYPES
+
+module Fib where
+
+data record R1 = R1 with
```

Comment:
this currently fails with 
```
            Malformed head of type or class declaration: record R1
```

that message comes from `checkTyClHdr` in `RdrHsSyn` (GHC); I thought the problem was that I added the `Parser.y` rule for flavored data types _after_ the one for regular data types but testing locally with those two switched gave me the same result. Maybe this will make more sense when I'm back